en/contributing.md

```diff
@@ -28,171 +30,261 @@ and listed here:
 https://github.com/CanonicalLtd/maas-docs/issues
 
 
-## How to contribute text
-
-Here are the basic steps necessary to get a change published on the website:
+## Contributing to a web page
 
-- [Fork the repository](https://help.github.com/articles/fork-a-repo) from
-  [github.com/CanonicalLtd/maas-docs](http://github.com/CanonicalLtd/maas-docs)
-- Make a local branch from your fork (and enter that branch)
-- Edit the source documents
-- Push your branch back to your fork on GitHub
-- [Submit a Pull Request](https://help.github.com/articles/creating-a-pull-request)
+Here is an overview of what's involved in getting a change published on the website:
 
-The source documents are located in the `src` directory. From there each
-language is separated into its own directory by language code. For instance,
-English is under `src/en`.
+1. [Fork][github-help-fork] the
+   [MAAS documentation repository][github-maas-docs]
+1. Clone that fork on your local system
+1. Create a branch from your local fork/clone
+1. Enter that branch and edit the source documents
+1. View the HTML locally
+1. Push your branch to your fork (on GitHub)
+1. [Create a Pull Request][github-help-pr] for that branch
 
-Once submitted, a Docs team member will review your work, suggest improvements,
-and eventually merge it with the master branch. Don't forget to review your
-work before submission!
+A Documentation team member will review your work, suggest improvements, and
+eventually merge it with the appropriate branch (series). Publication to the
+website is a separate step (performed internally), so it can be a few days
+before the changes actually show up. Please be patient!
 
 ### Metadata
 
-Each file has the potential to include metadata for various purposes. At the
-moment this is used to provide a title element, and also to implement a limited
-form of todo list items. Metadata is written as _key : value_ pairs AT THE VERY
-TOP of the document. E.g.
+Metadata can be included in any file. Currently, this is used for:
 
-```
-Title: Contributing to MAAS docs
-TODO: add section on metadata
-      spellcheck everything
+- title element
+- todo list (file improvements)
+- table of contents
+
+This information is written as key:value pairs **at the very top** of the
+page. For example:
+
+```no-highlight
+Title: Install from ISO | MAAS
+TODO: images need updating when Ubuntu 17.04 is released
+      check for changes to bug https://pad.lv/1625211 and modify text accordingly
+table_of_contents: True
 
 # Title of document
 
-Well written text goes here blah blah
+Text goes here blah blah blah
```

Comment:
You don't want it well-written? (Kidding, this is fine. The original needed a hyphen, the new text looks good.)